src/sidebarLearn.json

```
@@ -1,195 +1,195 @@
+      "sectionHeader": "APRENDA REACT"
     },
     {
-      "title": "Describing the UI",
+      "title": "Descrevendo a IU",
       "tags": [],
       "path": "/learn/describing-the-ui",
       "routes": [
         {
-          "title": "Your First Component",
+          "title": "Seu primeiro Componente",
           "path": "/learn/your-first-component"
         },
         {
-          "title": "Importing and Exporting Components",
+          "title": "Importando e Exportando Componentes",
           "path": "/learn/importing-and-exporting-components"
         },
         {
-          "title": "Writing Markup with JSX",
+          "title": "Escrevendo marcação com JSX",
           "path": "/learn/writing-markup-with-jsx"
         },
         {
-          "title": "JavaScript in JSX with Curly Braces",
+          "title": "JavaScript entre Chaves no JSX",
           "path": "/learn/javascript-in-jsx-with-curly-braces"
         },
         {
-          "title": "Passing Props to a Component",
+          "title": "Passando Propriedades a um Componente",
           "path": "/learn/passing-props-to-a-component"
         },
         {
-          "title": "Conditional Rendering",
+          "title": "Renderização condicional",
           "path": "/learn/conditional-rendering"
         },
         {
-          "title": "Rendering Lists",
+          "title": "Renderizando Listas",
           "path": "/learn/rendering-lists"
         },
         {
-          "title": "Keeping Components Pure",
+          "title": "Mantendo seu Componente Puro",
           "path": "/learn/keeping-components-pure"
         }
       ]
     },
     {
-      "title": "Adding Interactivity",
+      "title": "Adicionando interatividade",
       "path": "/learn/adding-interactivity",
       "tags": [],
       "routes": [
         {
-          "title": "Responding to Events",
+          "title": "Respondendo a Eventos",
           "path": "/learn/responding-to-events"
         },
         {
-          "title": "State: A Component's Memory",
+          "title": "Estado: Memória de um componente",
           "path": "/learn/state-a-components-memory"
         },
         {
-          "title": "Render and Commit",
+          "title": "Renderizar e Confirmar",
           "path": "/learn/render-and-commit"
         },
         {
-          "title": "State as a Snapshot",
+          "title": "Estado como um Snapshot",
           "path": "/learn/state-as-a-snapshot"
         },
         {
-          "title": "Queueing a Series of State Updates",
+          "title": "Enfileirando uma série de atualizações de estado",
           "path": "/learn/queueing-a-series-of-state-updates"
         },
         {
-          "title": "Updating Objects in State",
+          "title": "Atualizando objetos no estado",
           "path": "/learn/updating-objects-in-state"
         },
         {
-          "title": "Updating Arrays in State",
+          "title": "Atualizando matrizes no estado",
```

```suggestion
          "title": "Atualizando Vetores no Estado",
```

Quando vejo "arrays" sendo traduzido, geralmente é para "vetores", ao passo que "matrizes" geralmente são usadas para se referir a arrays N-dimensionais, com N > 1. O que você acha dessa alteração?

---

Eu sugiro a utilização da palavra 'Listas'. Vejo ela ser mais usada que vetores e matrizes.

---

> @natrodrigo mas listas são uma outra estrutura de dados bem definida. Acho que temos que tentar ser fiéis ao texto original nesse ponto

Mas não existe um objeto 'lista' no JS @jotaRenan. Não acho que isso gera um confluto

Então acho que o que mais faz sentido é usar a palavra Array mesmo então, porque na tradução do MDN eles não traduzem esse termo. Vetores e matrizes são mais utilizados em programação dentro da Engenharia em um contexto geralmente matemático.